Code/RDGeneral/CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/versions.h.cmake
 CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/RDConfig.h.cmake
               ${CMAKE_CURRENT_SOURCE_DIR}/RDConfig.h )
 
+FetchContent_MakeAvailable(better_enums)
+if(NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/enum.h)
+  file(COPY ${better_enums_SOURCE_DIR}/enum.h DESTINATION ${CMAKE_CURRENT_SOURCE_DIR})
+endif(NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/enum.h)
```

Comment:
I do not _have_ to copy file, but I prefer to do so to avoid annoying red underscores due to missing headers in IDEs such as Visual Studio Code, which won't find the header in the build directory, but will find it if copied in the source tree.